bootstrap/terraform/README.md

```diff
@@ -135,10 +135,6 @@ echo "$(kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.
 ## How to Destroy
 The following command destroys the resources created by `terraform apply`
 
-Step1: Delete resources created by Crossplane
-
-Step2: Terraform Destroy
-
```

Comment:
Since this is within the README for creating the EKS cluster, I thought cleaning up only what was made here would be within scope

Reply:
This is part of the crossplane-on-eks repo, so it is assumed you would deploy some Crossplane examples from the repo and those need to be cleaned up, otherwise the destroy script won't work.